lib/src/reddit.dart

```diff
@@ -347,6 +347,30 @@ class Reddit {
         siteName);
   }
 
+  static Reddit restoreInstalledAuthenticatedInstance(String credentialsJson,
```

Comment:
Can you add documentation for this? It can just be the relevant text from the documentation for `restoreAuthenticatedInstance`.